doc/autoscale/experiment/control_case1.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+DEFAULT_JOBNAME_PREFIX="mnist"
+
+function submit_general_job() {
```

Comment:
We can keep `submit_general_job`, but do we need to use it in the experiment?

Reply:
Following the TestCase1 of the [design](https://github.com/PaddlePaddle/cloud/blob/develop/doc/autoscale/experiment/autoscale_experiment.md#comparing-the-auto-scaling-training-job-and-the-general-training-job), we will compare the gernatl jobs and fault-tolerant jobs, so maybe we need this function?

Reply:
I see, sorry my mistake, I meant do we need the `faulttolerant` flag in the general job. But that is a minor thing, don't worry about it.